vmm/src/config.rs

```diff
@@ -20,6 +20,12 @@ use virtio_devices::{RateLimiterConfig, TokenBucketConfig};
 
 pub const DEFAULT_VCPUS: u8 = 1;
 pub const DEFAULT_MEMORY_MB: u64 = 512;
+
+// When booting with PVH boot the maximum physical addressable size
+// is a 46 bit address space even when the host supports with 5-level
+// paging.
+pub const DEFAULT_MAX_PHYS_BITS: u8 = 46;
```

Comment:
Question: Can you please explain why we need to set default `max_phys_bit` to 46, given 4-level paging allows 48 bits virtual address?

Reply:
https://github.com/torvalds/linux/commit/ed7588d5dc6f5e7202fb9bbeb14d94706ba225d7#diff-9e839b4f67eea9775eeced95f8cfb185806c89f1071e6afb710a174c093ec209R56